test/dlc_tests/sanity/test_telemetry.py

```diff
@@ -27,6 +27,7 @@ def test_telemetry_bad_instance_role_disabled_cpu(cpu, ec2_client, ec2_instance,
 @pytest.mark.processor("neuron")
 @pytest.mark.integration("telemetry")
 @pytest.mark.parametrize("ec2_instance_type", ["inf1.xlarge"], indirect=True)
+@pytest.mark.skip("Feature doesn't exist on Neuron DLCs")
```

Comment:
is this a placeholder if we are skipping the test for neuron?

Reply:
Yes, this is a placeholder for when it is added in to the framework source code.